src/components/Users/UserAvailabilityTab.tsx

```
@@ -0,0 +1,330 @@
+                      month: "short",
+                      year: "numeric",
+                    })}
+                  </p>
+                  <Button variant="outline" size="sm">
+                    Add Exception
+                  </Button>
+                </div>
+
+                <ScrollArea className="h-[22rem]">
+                  {templates.map((template) => (
+                    <div className="border-t pt-3 mt-3">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)